pkg/controller/vm_webhook.go

```diff
@@ -619,3 +640,12 @@ func ValidateMultusNetworkSource(ctx context.Context, source *virtv1alpha1.Multu
 	}
 	return errs
 }
+
+func generateMAC() (net.HardwareAddr, error) {
+	prefix := []byte{0x52, 0x54, 0x00}
```

Comment:
If my memory serves me right, this prefix belongs to QEMU? Should we use another prefix?

Reply:
yes, this prefix is for QEMU,  but I didn't find a canonical place to get a mac address OUI, maybe we can use any legal unicast prefix.

Reply:
Found an un-verified list of vendors: https://gist.github.com/aallan/b4bb86db86079509e6159810ae9bd3e4 . It shows that this prefix actually belongs to Realtek. I think it's safe to use this prefix